backend/makefile.in

```diff
@@ -53,42 +55,31 @@ $(JAR): $(SRC)
 
 # Taxons and Lineages {{{ ------------------------------------------------------
 <<<TAXDIR>>>/taxdmp.zip:
-	echo "Starting taxon dump download."
+	<<<LOGADD>>> "Started taxon dump download."
 	mkdir -p <<<TAXDIR>>>
 	-rm -f $@
 	wget --no-verbose "<<<TAXON_URL>>>" -O $@
-	echo "Finished taxon dump download."
+	<<<LOGADD>>> "Finished taxon dump download."
 
 <<<INTDIR>>>/names.dmp <<<INTDIR>>>/nodes.dmp: <<<TAXDIR>>>/taxdmp.zip
-	echo "Starting unzipping names and nodes from the taxon dump."
+	<<<LOGADD>>> "Started unzipping names and nodes from the taxon dump."
 	<<<CMD_UNZIP>>> -o $< $(notdir $@) -d $(dir $@)
-	echo "Finished unzipping names and nodes from the taxon dump."
```

Comment:
was this removed by you or by a merge commit?

Reply:
I might have wrongly merged this. Fixed in e3c8bb5

Reply:
interpro entries & info have been added to pept2funt & peptinfo pages in commits 
c6a891e & 7d41769